lib/engines/content_object_store/test/integration/content_block_editions_test.rb

```diff
@@ -45,5 +46,15 @@ class ContentBlockEditionsTest < ActionDispatch::IntegrationTest
         details:,
       },
     }
+
+    assert_equal initial_document_count + 1, ContentObjectStore::ContentBlockDocument.count, "Expected ContentObjectStore::ContentBlockDocument count to increase by 1"
```

Comment:
Though we include `Identifiable` to sort this behaviour for us and that is presumably tested, I think we ought to have some coverage and documentation on this too? Rather than stubbing `create!` we let it run all the way through.

Reply:
I think we can use [`assert_changes`](https://apidock.com/rails/ActiveSupport/Testing/Assertions/assert_changes) here instead of recording the initial counts?

Reply:
I'll definitely check that out, doing counts was feeling too verbose.